addons/website/static/src/js/editor/widget_link.js

```diff
@@ -40,29 +43,71 @@ weWidgets.LinkTools.include({
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * @override web_editor - link.js abstract method
+     * @param {string} url
+     * @returns {boolean} Whether or not link should open in a new page
+     * @private
```

Comment:
```suggestion
     * @override
```
is enough, as far as I know